TLSharp.Core/DataCenterIPVersion.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace TLSharp.Core
+{
+    /// <summary>
+    /// When the Telegram server responds with a set of addresses to connect to, DataCenterIPVersion indicates a preference 
+    /// for how to choose the IP address to connect to 
+    /// </summary>
+	public enum DataCenterIPVersion
+    {
+        /// <summary>
+        /// Prefers IPv6 addresses if any is passed by Telegram
+        /// </summary>
+        Default  =   0,  
+        /// <summary>
+        /// Takes only IPv4 addresses
+        /// </summary>
+        OnlyIPv4 =   1,
+        /// <summary>
+        /// Takes only IPv6 addresses
+        /// </summary>
+        OnlyIPv6 =   2,
+        /// <summary>
+        /// Connection to IPv4 addresses is preferred to IPv6 addresses
+        /// </summary>
+        PreferIPv4 = 3,
+        /// <summary>
+        /// Connection to IPv6 addresses is preferred to IPv6 addresses
```

Comment:
typo here ^

Reply:
strange, i changed it O.o

Reply:
yes, this one was fixed, but I still see the text “Prefers IPv6 addresses if any is passed by Telegram” describing “Default”

Reply:
it's what actually does

Reply:
No, the text “Prefers IPv6 addresses if any is passed by Telegram” should describe "PreferIPv6", not "Default".

Reply:
remember Default doesn't do any filtering, just chooses the first datacenter's IP given by telegram servers

Reply:
it doesn't do any filtering, but it's a side effect of the algorithm to pick an ipv6 if any is passed just because it will be at the end of the list.

anyways, because the default behavious might change in the future, we can remove this comment.

Reply:
we can fix the side-effect, don't do any OrderBy if .Default is chosen please.